examples/src/RecordMovieComponent.tsx

```diff
@@ -43,18 +45,25 @@ const RecordMovieComponent = ({
         <div>
             <button
                 onClick={startRecording}
-                disabled={isRecording || !browserSupported}
-            >
+                disabled={
+                    !isRecordingEnabled || isRecording || !browserSupported
+                }>
                 Start Recording
             </button>
             <button
                 onClick={stopRecording}
-                disabled={!isRecording || !browserSupported}
-            >
+                disabled={
+                    !isRecordingEnabled || !isRecording || !browserSupported
+                }>
```

Comment:
I know this is the testbed example, but having two triple ORs that only differ by one is confusing to read. I would suggest figuring out a way to make this clearer. if recording is not enabled I would think this whole component wouldn't render? Instead of just disabling the buttons